spell/spell-reviewer-mainnet-checklist.md

```diff
@@ -1,369 +1,372 @@
-      * [ ] IF addresses not PR'ed in by the SubDAO team (use git blame for example), SubDAO content addresses all have inline comment for provenance or source being OKed by SubDAO
+    * [ ] IF addresses not PR'ed in by the SubDAO team (use git blame for example), SubDAO content addresses all have inline comment for provenance or source being OKed by SubDAO
     * [ ] SubDAO actions match Exec Doc (only where inline with main spell code) and do not affect core contracts
     * [ ] Core contract knock-on actions (such as offboarding or setting DC to 0) are present in the exec and match the code
     * [ ] External calls for SubDAO content are NOT delegate call
     * [ ] Code does not have untoward behavior within the scope of Maker Core Contracts (e.g. up to the SubDAO proxy)
-* [ ] External Contracts Calls (Not SubDAOs, e.g. Starknet)
-  * [ ]  Target Contract don't block spell execution
-  * [ ]  External call is NOT delegate call
-  * [ ]  Target Contract doesn't have permissions on the Vat
-  * [ ]  Target Contract doesn't do anything untoward (e.g. interacting with unsafe contracts)
-  * [ ]  Contracts deployed via `CREATE2` (e.g. if it looks like a vanity address) do not have `selfdestruct` in their code
-  * [ ]  MCD Pause Proxy doesn't give any approvals
-  * [ ]  All possible actions of the Target Contract are documented
-  * [ ]  Target contract is not upgradable
-  * [ ]  Target Contract is included in the ChainLog
-  * [ ]  Test Coverage is comprehensive
-* [ ] ChainLog
-  * [ ] Increment ChainLog version based on update type
-    * [ ] Major -> New Vat (++.0.0)
-    * [ ] Minor -> Core Module (DSS) Update (e.g. Flapper) (0.++.0)
-    * [ ] Patch -> Collateral addition or addition/modification (0.0.++)
-* [ ] `addresses_mainnet.sol` matches spell code
+* IF external contracts calls present (Not SubDAOs, e.g. Starknet)
+  * [ ] Target Contract don't block spell execution
+  * [ ] External call is NOT delegate call
+  * [ ] Target Contract doesn't have permissions on the Vat
+  * [ ] Target Contract doesn't do anything untoward (e.g. interacting with unsafe contracts)
+  * [ ] Contracts deployed via `CREATE2` (e.g. if it looks like a vanity address) do not have `selfdestruct` in their code
+  * [ ] MCD Pause Proxy doesn't give any approvals
+  * [ ] All possible actions of the Target Contract are documented
+  * [ ] Target contract is not upgradable
+  * [ ] Target Contract is included in the ChainLog
+  * [ ] Test Coverage is comprehensive
+* IF spell interacts with ChainLog
+  * [ ] ChainLog version is incremented based on update type
+    * Major -> New Vat (++.0.0)
+    * Minor -> Core Module (DSS) Update (e.g. Flapper) (0.++.0)
+    * Patch -> Collateral addition or addition/modification (0.0.++)
+  * [ ] New addresses are added to the `addresses_mainnet.sol`
+  * [ ] Changes are tested via `testNewOrUpdatedChainlogValues`
```

Comment:
This might be not required anymore as per latest changes in [#374](https://github.com/makerdao/spells-mainnet/pull/374)
```suggestion
```

Reply:
Yeah, but let's add this change via a separate tiny PR